packages/react/src/primitives/Menu/__tests__/menu.test.tsx

```diff
@@ -161,5 +207,85 @@ describe('Menu', () => {
 
       expect(disabled).toHaveClass('amplify-button--disabled');
     });
+
+    it('should add the Amplify UI Button disabled class to when Menu is disabled', async () => {
```

Comment:
We can simplify the test description here to only include the behavioral assertion:
```suggestion
    it('has the expected class when disabled and open', async () => {
```